Documentation/getting-started.md

```diff
@@ -0,0 +1,82 @@
+# Getting Started with Analytics SDK
+
+## Before starting
+
+Analytics extension has a dependency on [AEP Core SDK and AEP Identity SDK](https://github.com/adobe/aepsdk-core-android#readme) which must be installed to use the extension.
+
+## Add Analytics extension to your app
+
+1. Installation via [Maven](https://maven.apache.org/) & [Gradle](https://gradle.org/) is the easiest and recommended way to get the AEP SDK into your Android app. Add a dependency on Analytics and Core to your mobile application. To ensure consistent builds, it is best to explicitly specify the dependency version and update them manually.
+
+   ```
+   implementation 'com.adobe.marketing.mobile:core:2.+'
+   implementation 'com.adobe.marketing.mobile:identity:2.+'
+   implementation 'com.adobe.marketing.mobile:analytics:2.+'
+   ```
+
+> **Warning**  
+> Using dynamic dependency versions is not recommended for production apps. Refer to this [page](https://github.com/adobe/aepsdk-core-android/blob/main/Documentation/MobileCore/gradle-dependencies.md) for managing Gradle dependencies.
+
+2. Import MobileCore, Identity and Analytics extensions:
+
+   ### Java
+
+   ```java
+   import com.adobe.marketing.mobile.MobileCore;
+   import com.adobe.marketing.mobile.Identity;
+   import com.adobe.marketing.mobile.Analytics;
+   ```
+
+   ### Kotlin
+
+   ```kotlin
+   import com.adobe.marketing.mobile.MobileCore
+   import com.adobe.marketing.mobile.Identity
+   import com.adobe.marketing.mobile.Analytics
+   ```
+
+3. Import the Analytics library into your project and register it with `MobileCore`
+
+   ### Java
+
+   ```java
+   public class MainApp extends Application {
+        private static final String APP_ID = "YOUR_APP_ID";
```

Comment:
nit: can we rename to `private final String ENVIRONMENT_FILE_ID = "YOUR_APP_ENVIRONMENT_ID";` for both java/kotlin